polar_connector:
  ros__parameters:
    publish_acceleration: True
    publish_hr: True
    publish_ecg: True
    bluetooth_adapter: "hci1"
